catalogue/config.py

```diff
@@ -0,0 +1,122 @@
+import os.path
+import csv
+import argparse
```

Comment:
Do you use this module, or just `namespace` (which you import in the next line)?